src/styles/blocks/card.scss

```
@@ -0,0 +1,101 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+}
```

Move font family to the html selector
```suggestion
html {
  font-family: Roboto, sans-serif;
}
```